### ISSUE TITLE
UE55H6400 wrong work

### ISSUE THREAD

**Issue by EvGeniyLell** [mmende/homebridge-samsungtv-control2#64]
after then as I pairing (by pair1 method) my device successfully
(no one else method of pairing does not work for me, i had a pairing error)
and plugin successfully send mute to my tv
i have some issues:
1) button POWER in widget don't turn off my tv
2) also INPUTS don't work as expected 
 { // it turn on/off "FOOTBALL" mode
   "name": "HDMI 1",
   "keys": "KEY_HDMI1"
}, 
{ // it do nothing 
   "name": "HDMI 2",
   "keys": "KEY_HDMI2"
},

how i can fix it?
i will be grateful for the help

**Comment by mmende** [comment]
Your TV might not support the keys `hdmi1` and `hdmi2`... unfortunately samsung's inofficial api is pretty inconsistent across devices. Can you try the `hdmi` key (without number)? This might also be the reason for not being able to turn off the tv, the library tries to find out if your tv is turned off with `KEY_POWER` or `KEY_POWEROFF` (in your case it uses `power` as you have a H-Series as far as I remember). Maybe your tv has to be turned off with `power` instead...

**Comment by EvGeniyLell** [comment]
yes you are right, the `KEY_HDMI` key works for my TV, it switches the HDMI inputs one by one, thanks!
but I still have a problem with the big POWER button
maybe you have another solution to fix this?
as an example
```{
     "platform": "SamsungTVControl",
     "devices": [
         {
             "usn": "uuid: ---",
             "token": "eyJz ---",
             "name": "TV Sumsung 55",
             "customKeyForPowerButton": "KEY_POWER", // <- some like this
             "inputs": [
                 {
                     "name": "HDMI",
                     "keys": "KEY_HDMI"
                 },
                 {
                     "name": "MUTE",
                     "keys": "KEY_MUTE"
                 },
                 {
                     "name": "POWER",
                     "keys": "KEY_POWER"
                 }
             ]
         }
     ]
}

**Comment by EvGeniyLell** [comment]
and, if it posible, can i configurate remout control for recognises hold `volume` buttons?
now volume changed up/down on 1 point only when i release `volume` button. holding has not effect. can we fix/improve it?
